### DISCUSSION TITLE
I found some apps that can't be decrypted and used.

### DISCUSSION THREAD

**Discussion by daoba2733** [PlayCover/PlayCover#1114] [Q&A] [answered]
Let me redescribe my problem.
After some apps are decrypted into IPA format, any way to install them will prompt illegal programs, but the IPA exported by imazing is installed using a jailbroken device, and it can be used normally.
Concluding that I wish I could install undecrypted IPA on my mac.

**Comment by daoba2733** [comment]


**Reply by daoba2733** [reply]
plz。help me

**Comment by zanderp25** [comment]
Unfortunately, PlayCover itself cannot decrypt IPAs. It's not possible to decrypt IPAs except by installing them on an iOS device, due to the way Apple has it set up. The only way I know to get a decrypted IPA is to use some jailbreak method on an iOS device. I'm not experienced with that method so I'm not able to give you advice on how to decrypt IPAs.

Also, what do you mean by "prompt illegal programs"?

**Reply by daoba2733** [reply]
For example, this South Korean app will prompt for illegal programs after decrypting the obtained iPad, regardless of whether it is installed on any device.
If I use IMAZING to export an IPA, that is, to install it without decryption. The app will not prompt for illegality.

I heard that the m1 device installed with the 11.2.3 Mac can directly use an unencrypted IPA, but unfortunately, I purchased m2

**Reply by daoba2733** [reply]
https://github.com/PlayCover/PlayCover/assets/146572512/61f8be49-4448-4b8a-9fc1-1abcc8b1a7c0

**Reply by zanderp25** [reply]
Ah, I see what's going on here. This program probably has an anti-cheat detection tool which is probably being triggered by PlayTools. You could try uninstalling PlayTools, but you would not be able to use the keymapping feature. Even after uninstalling PlayTools, I don't guarantee it will work, either.

**Reply by daoba2733** [reply]
It's not because PlayTools triggered the anti-cheat system.
This is the case even if I install on ios.
But the ipa I exported using imazing is normal. Hope to support imazing.

**Reply by daoba2733** [reply]
Developers who can do anything. May the Lord bless you

**Reply by daoba2733** [reply]
[https://drive.google.com/file/d/1RYvoyjs7pFJLqTcteKX9Pr0U2sve2l3n/view?usp=share_link](url)I uploaded this ipa

**Reply by zanderp25** [reply]
I can't get the same thing you are getting. All I am getting is a message saying that the game is not available in my region. I don't live in South Korea, and I don't know how I could get around this message.

**Reply by daoba2733** [reply]
Yes, you need to use a Korean proxy to pass this game's server verification.
I used [雷神加速器](https://apps.apple.com/cn/app/%E9%9B%B7%E7%A5%9E%E6%89%8B%E6%B8%B8%E5%8A%A0%E9%80%9F%E5%99%A8-%E4%B8%93%E6%B3%A8%E4%BA%8E%E6%B8%B8%E6%88%8F%E5%8A%A0%E9%80%9F-%E7%95%85%E7%8E%A9%E6%B5%B7%E9%87%8Fsteam%E6%B8%B8%E6%88%8F/id1613506145) to bypass Korean regional verification
[雷神加速器](https://apps.apple.com/cn/app/%E9%9B%B7%E7%A5%9E%E6%89%8B%E6%B8%B8%E5%8A%A0%E9%80%9F%E5%99%A8-%E4%B8%93%E6%B3%A8%E4%BA%8E%E6%B8%B8%E6%88%8F%E5%8A%A0%E9%80%9F-%E7%95%85%E7%8E%A9%E6%B5%B7%E9%87%8Fsteam%E6%B8%B8%E6%88%8F/id1613506145) can be downloaded from the appstore
I also decrypted the ipa but it seems that he can no longer create vpn access on the mac, but VPN access can be used for downloads via the APPstore

Maybe the decrypted ipa can't mobilize VPN privileges,

[This is the video. ](https://drive.google.com/file/d/1Ao4-a-jprATUvscdom9mYN_MkQ_oaS-C/view)

[雷神加速器.ipa ](https://drive.google.com/file/d/1NX12sqRdoieEyWzHyP47-eVddgMIuPi_/view)

**Reply by daoba2733** [reply]
[This is how I opened it..](https://drive.google.com/file/d/14v1dQ-4jBzF5Y2GgL0ZV8xu7RtU6fvpe/view?usp=share_link)

**Reply by daoba2733** [reply]
Dear developer, how is it going?

**Reply by zanderp25** [reply, marked as answer]
Unfortunately, the VPN did not work for me and I no longer have time to investigate this issue.

**Reply by daoba2733** [reply]
thanks

**Comment by MoeNyanSakura** [comment]
没有砸壳 IPA包？ 晚上有一些是没有砸壳的

**Reply by daoba2733** [reply]
Without decrypted IPA, palycover cannot be used.
